vendor/deadline/custom/plugins/GlobalJobPreLoad.py

```diff
@@ -55,21 +58,32 @@ def inject_openpype_environment(deadlinePlugin):
                   "AVALON_TASK, AVALON_APP_NAME"
             raise RuntimeError(msg)
 
-        print("args:::{}".format(args))
+        if not os.environ.get("OPENPYPE_MONGO"):
+            print(">>> Missing OPENPYPE_MONGO env var, process won't work")
+
+        env = os.environ
+        env["OPENPYPE_HEADLESS_MODE"] = "1"
+        env["AVALON_TIMEOUT"] = "5000"
 
-        exit_code = subprocess.call(args, cwd=os.path.dirname(openpype_app))
-        if exit_code != 0:
-            raise RuntimeError("Publishing failed, check worker's log")
+        print(">>> Executing: {}".format(args))
+        std_output = subprocess.check_output(args,
+                                             cwd=os.path.dirname(openpype_app),
+                                             env=env)
+        print(">>> Process result {}".format(std_output))
```

Comment:
Now I am not sure if `format()` won't actually take care of it but as I reacall, output from this comand is bytes so it needs to be properly encoded to utf-8?

```suggestion
        print(">>> Process result {}".format(std_output.decode("utf-8")))
```

Reply:
Its getting printed even without this, but the content length is imho limited.